Library/Formula/objective-caml.rb

```diff
@@ -1,9 +1,23 @@
+# OCaml does not preserve binary compatibility across compiler releases,
+# so when updating it you should ensure that all dependent packages are
+# also updated by incrementing their revisions.
```

Comment:
The only one that we can do is a local OPAM bootstrap.  I'll take a look at this.